contracts/StablecoinConverter.sol

```diff
@@ -499,7 +499,7 @@ contract StablecoinConverter is EpochTokenLocker {
             getBalance(user, tokenIdToAddressMap(order.sellToken))
```

Comment:
The doc comments should be updated with the new equation.

Reply:
fixed.... weird place to comment though...

Reply:
It didn’t let me comment on the actual line :( that’s as far up it went